pkg/activator/net/revision_backends.go

```diff
@@ -264,7 +280,7 @@ func (rw *revisionWatcher) checkDests(curDests, prevDests dests) {
 
 	// If we have discovered that this revision cannot be probed directly
 	// do not spend time trying.
-	if rw.podsAddressable {
+	if rw.podsAddressable || rw.usePassthroughLb {
```

Comment:
I think this isn't possible since if usePassthroughLb is set we will never try service scraping, and therefore [would never mark podAddressable false](https://github.com/knative/serving/blob/a18955aa7d5059f6b2126a1a27b3b7a9ec38ae8a/pkg/activator/net/revision_backends.go#L348).

Reply:
Fair enough, I still feel like it's worth having it here for added clarity?

Reply:
Maybe, fwiw it was more confusing to me on first reading: ie even if `usePassthroughLb` is set it wouldnt make much sense to try to directly access pods if we already determined they're not accessible. Then I realised this never happens anyway. Also if we do want to keep it we should probably update the comment to match.

Reply:
I'll drop it then.